apps/docs/pages/guides/getting-started/quickstarts/nextjs.mdx

```diff
@@ -100,7 +100,7 @@ export const meta = {
           const { data: countries } = await supabase.from("countries").select();
 
           return (
-            <ul className="my-auto">
+            <ul className="my-auto text-white">
```

Comment:
I don't know if this will work on light mode?

Reply:
I think there is some sort of `brand-` class we can use insted

Reply:
You were right @kiwicopple, `text-white` would have displayed white text on white background in light mode. The correct class is `text-foreground`. The `foreground` and `background` colours are defined in the `tailwind.config.js` file. Now it works for both light and dark modes.